### DISCUSSION TITLE
Focus not Working

### DISCUSSION THREAD

**Discussion by caiocs** [sceneview/sceneform-android#199] [General]
Hello, 
I was testing your samples and the Augmented Images was without the auto focus. I put the config.setFocusMode(Config.FocusMode.AUTO) inside the onSessionConfiguration method. But the camera still without the auto focus. Am I doing something wrong?

**Comment by ThomasGorisse** [comment]
ARCore open bug here: https://github.com/google-ar/arcore-android-sdk/issues/1300

**Comment by caiocs** [comment]
Hey @ThomasGorisse, I found a work around. I don't know why if you left the activity and come back the focus start to work. I'm opening a new activity and closing immediately (the user can't see the activity opening) with that the focus starts to work.
